src/NUnitFramework/framework/Attributes/SetUpAttribute.cs

```diff
@@ -26,8 +26,7 @@ namespace NUnit.Framework
     using System;
 
     /// <summary>
-    /// Attribute used to identify a method that is called 
-    /// immediately before each test is run.
+    /// Identifies a method to be called prior to the running of each test.
```

Comment:
I prefer "Identifies a method to be called immediately before each test is run." to match the TearDownAttribute.